.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -10,23 +10,66 @@ assignees: ''
 
 The release number or commit hash of the version you're using.
 
-## Operating system & version
+## \[Optional\] Setup information
```

Comment:
Maybe “Platform information”? Setup is a bit overloaded, in my opinion. I’ve taken to using platform for CPU/GPU/graphics API/OS.

Reply:
Yeah, I'm not happy with the wording either. Your suggestion feels too limited though 🤔

Reply:
Really I want "User-specific Information", but I'm really not sure how to make that clear enough.

Reply:
I would do:

> ## Relevant System Information

If relevant to your issue, what is your operating system / distro, hardware (CPU/GPU), drivers, etc…?

Reply:
```suggestion
## \[Optional\] Relevant system infromation
```